### DISCUSSION TITLE
Yarn Vite debug how to?

### DISCUSSION THREAD

**Discussion by meglio** [remix-run/remix#8942] [Q&A]
The documentation misses details on how to debug a yarn managed project when using Vite.

What the documentation says for Vite:

In `package.json`,  set `"dev": "remix vite:dev"`.

To debug, run: `NODE_OPTIONS="--inspect-brk" npm run dev`

Now, what Yarn documentation says for the "run" command:

> --inspect-brk | Forwarded to the underlying Node process when executing a binary

... and gives an example: `yarn run --inspect-brk webpack`

---

Now, the end goal is to have a single command in `package.json` like this:

`"debug": "... something ..."`

So that I can either use `yarn dev` or `yarn debug` in console and not worry about all those parameters etc. But it seems like there is no clear path to that?

**Comment by lukewilson2002** [comment]
@meglio Did you ever figure this out?
